apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/Components/BuildDisplayItem.tsx

```diff
@@ -328,22 +332,18 @@ function ExcludeButton({ id }: { id: string }) {
       title={
         <Box>
           <Typography>{t`excludeArt.excludeArtifactTip`}</Typography>
-          <Typography>
-            <ColorText color={excluded ? 'error' : 'success'}>
-              {t(excluded ? 'excludeArt.excluded' : 'excludeArt.included')}
-            </ColorText>
-          </Typography>
         </Box>
       }
       placement="top"
       arrow
     >
       <Button
         onClick={toggle}
-        color={excluded ? 'error' : 'success'}
+        color={excluded ? 'secondary' : 'success'}
         size="small"
+        startIcon={excluded ? <CheckBoxOutlineBlankIcon /> : <CheckBoxIcon />}
       >
-        {excluded ? <BlockIcon /> : <ShowChartIcon />}
+        <span>{t`excludeArt.included`}</span>
```

Comment:
```suggestion
        <span>{excluded ?t`excludeArt.included` : t`excludeArt.included`}</span>
```

Reply:
Why the ternary if it'll end up displaying the same text regardless of the value of `excluded`?

Reply:
Opps, copying error on my end. I was refering to https://github.com/frzyc/genshin-optimizer/blob/master/libs/gi/localization/assets/locales/en/page_character_optimize.json#L37
which is 
```
t`excludeArt.excluded`
```

Reply:
I did consider using `excludeArt.excluded`, but using `excludeArt.included` instead as the text means the button's behavior better matches the behavior of the character and set effect exclusion buttons (green if included, gray if excluded).

Though I could use the excluded text and invert the colors so green is excluded and gray is included. It seems a little weird in my head, but seeing as that's how the Exclude Artifacts button works anyway, it's consistent on that end.